docker/distelli-manifest.yml

```diff
@@ -0,0 +1,9 @@
+release-engineering/ezbake:
```

Comment:
Is this something we want, now that Pipelines/Distelli is EOL?

Reply:
Yes. We haven't figured out our migration path yet for pupperware, and will migrate everything once we figure that out. I don't want that to block other containerization work

Reply:
that's fair, just wanted to check 😊